vnext/Microsoft.ReactNative/ABIModule.cpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "ABIModule.h"
+#include "NativeModulesProvider.h"
+
+namespace winrt::Microsoft::ReactNative::Bridge {
+
+auto ABIModule::getName() -> std::string {
+  auto name = m_module.Name();
+  return to_string(name);
+}
+
+auto ABIModule::getConstants() -> std::map<std::string, folly::dynamic> {
+  std::map<std::string, folly::dynamic> constants;
+  auto constantsMapView = m_module.Constants();
```

Comment:
[nit] Using an explicit type instead of 'auto' would help inexperienced WinRT devs (like me :) to better understand what's going on.

Reply:
Good feedback.  I wasn't 100% consistent in my style.  Sorry!

Reply:
Switching this to be IMapView<hstring, IInspectable> as part of my upcoming samples PR.